core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerUtils.scala

```diff
@@ -144,6 +144,11 @@ private[mesos] trait MesosSchedulerUtils extends Logging {
     registerLatch.countDown()
   }
 
+  protected def markErr(): Unit = {
+    println(">>>>>>>>> markErr is called")
```

Comment:
Should this be a `logError` instead? Or simply nothing, since it looks like the error is logged on all paths to this method.